Application/src/Utils/Structure.js

```diff
@@ -1,64 +1,62 @@
 import { Image } from 'image-js';
 import { changeDpiDataUrl } from 'changedpi';
 import { getText } from './Text';
-import { getCardHeaderUtilityClass } from '@mui/material';
```

Comment:
We need our next refactor PR needs to be a refactor that removes all unused dependencies.

Reply:
For some reason, that import keeps getting automatically included without me noticing. Will remove.

Reply:
Fair